src/Resources/app/storefront/dist/storefront/js/fin-search.js

```diff
@@ -1 +1 @@
:t.dataset.label,id:t.id})})):e=[],e}},{key:"setValuesFromUrl",value:function(){var t=this,e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:{},n=!1,r=e[this.options.name],o=r?r.split("|"):[],a=this.selection.filter((function(t){return!o.includes(t)})),s=o.filter((function(e){return!t.selection.includes(e)}));return(a.length>0||s.length>0)&&(n=!0),s.forEach((function(e){var n=i.a.querySelector(t.el,'[id="'.concat(e,'"]'),!1);n&&(n.checked=!0,t.selection.push(n.id))})),a.forEach((function(e){t.reset(e),t.selection=t.selection.filter((function(t){return t!==e}))})),this._updateCount(),n}},{key:"_onChangeFilter",value:function(){this.listing.changeListing(!0,{p:1})}},{key:"reset",value:function(t){var e=i.a.querySelector(this.el,'[id="'.concat(t,'"]'),!1);e&&(e.checked=!1)}},{key:"resetAll",value:function(){this.selection.filter=[];var t=i.a.querySelectorAll(this.el,"".concat(this.options.checkboxSelector,":checked"),!1);t&&r.a.iterate(t,(function(t){t.checked=!1}))}},{key:"refreshDisabledState",value:function(t){var e=t[this.options.name];!e.entities||e.entities.length<1?this.disableFilter():(this.enableFilter(),this._disableInactiveFilterOptions(e.entities.map((function(t){return t.id}))))}},{key:"_disableInactiveFilterOptions",value:function(t){var e=this,n=i.a.querySelectorAll(this.el,this.options.checkboxSelector);r.a.iterate(n,(function(n){!0!==n.checked&&(t.includes(n.id)?e.enableOption(n):e.disableOption(n))}))}},{key:"disableOption",value:function(t){var e=t.closest(this.options.listItemSelector);e.classList.add("disabled"),e.setAttribute("title",this.options.snippets.disabledFilterText),t.disabled=!0}},{key:"enableOption",value:function(t){var e=t.closest(this.options.listItemSelector);e.removeAttribute("title"),e.classList.remove("disabled"),t.disabled=!1}},{key:"enableAllOptions",value:function(){var t=this,e=i.a.querySelectorAll(this.el,this.options.checkboxSelector);r.a.iterate(e,(function(e){t.enableOption(e)}))}},{key:"disableFilter",value:function(){var t=i.a.querySelector(this.el,this.options.mainFilterButtonSelector);t.classList.add("disabled"),t.setAttribute("disabled","disabled"),t.setAttribute("title",this.options.snippets.disabledFilterText)}},{key:"enableFilter",value:function(){var t=i.a.querySelector(this.el,this.options.mainFilterButtonSelector);t.classList.remove("disabled"),t.removeAttribute("disabled"),t.removeAttribute("title")}},{key:"_updateCount",value:function(){this.counter.innerText=this.selection.length?"(".concat(this.selection.length,")"):""}}])&&c(n.prototype,o),a&&c(n,a),e}(o.a);y=v,b="options",d=s()(o.a.options,{checkboxSelector:".filter-multi-select-checkbox",countSelector:".filter-multi-select-count",listItemSelector:".filter-multi-select-list-item",snippets:{disabledFilterText:"Filter not active"},mainFilterButtonSelector:".filter-panel-item-toggle"}),b in y?Object.defineProperty(y,b,{value:d,enumerable:!0,configurable:!0,writable:!0}):y[b]=d}},[["9TfS","runtime","vendor-node","vendor-shared"]]]);
```

Comment:
:information_source: Only a newline at the end was added, as my git configuration automatically adds newlines to the end of all files, which is neat as I do not need to think about adding those myself.